applications/accounting/src/main/groovy/org/apache/ofbiz/accounting/AutoAcctgFinAccountTests.groovy

```diff
@@ -179,20 +178,26 @@ class AutoAcctgFinAccountTests extends OFBizTestCase {
         assert serviceResult.finAccountAuthId != null
     }
 
-    void setFinAccountTransStatus() {
+    void testSetFinAccountTransStatus() {
         Map serviceCtx = [
                 finAccountTransId: '1010',
                 statusId: 'FINACT_TRNS_APPROVED',
                 userLogin: userLogin
         ]
+        GenericValue finAccountTrans = from('FinAccountTrans')
+                .where('finAccountTransId', '1010')
+                .queryOne()
+        String oldStatusId = finAccountTrans.statusId
+
         Map serviceResult = dispatcher.runSync('setFinAccountTransStatus', serviceCtx)
         assert ServiceUtil.isSuccess(serviceResult)
 
-        GenericValue finAccountTrans = from('FinAccountTrans')
+        finAccountTrans = from('FinAccountTrans')
                 .where('finAccountTransId', '1010')
                 .queryOne()
         assert finAccountTrans
         assert finAccountTrans.statusId == 'FINACT_TRNS_APPROVED'
-        assert finAccountTrans.oldStatusId == 'FINACT_TRNS_CREATED'
```

Comment:
Appears to be a change in behaviour.

Reply:
Oh I remember this one, this was a test that was implemented but was never played due to its naming.
I fixed that by renaming the `setFinAccountTransStatus` method to `testSetFinAccountTransStatus`.
Result the test was failing, i guessed that the service behind used to send back oldStatusId, but was modified in the past. 
To make the test pass, i extracted lin 190 the status before calling service. Hence the removal of the map reference you detected.
If that is ok we can resolve the discussion :)